cluster/terraform_kubernetes/variables.tf

```diff
@@ -186,6 +186,7 @@ variable "alertable_apps" {
 }
 
 locals {
+  alertmanager_external_url = module.cluster_data.ingress_domain
```

Comment:
It doesn't look like it's used. Use it in --web.external-url or delete it

Reply:
removed